xds/src/test/java/io/grpc/xds/XdsClientImplTest.java

```diff
@@ -2673,6 +2710,29 @@ private static Cluster buildCluster(String clusterName, @Nullable String edsServ
     return clusterBuilder.build();
   }
 
+  private static Cluster buildSecureCluster(String clusterName, @Nullable String edsServiceName,
+      boolean enableLrs, @Nullable UpstreamTlsContext upstreamTlsContext) {
```

Comment:
`upstreamTlsContext` should be not null (in this case you could do `buildCluster(...).toBuilder().setTlsContext(...).build()`. 
or, 
keep it `@Nullable`, and `buildCluster` calls `buildSecureCluster` with null `upstreamTlsContext`.

Reply:
Now buildCluster calls buildSecureCluster with null upstreamTlsContext.